src/test/java/com/epam/tc/hw2/ex1/GithubPageSeleniumTestsEx1.java

```diff
@@ -0,0 +1,95 @@
+package com.epam.tc.hw2.ex1;
+
+import com.epam.tc.hw2.WebdriverSettings;
+import java.util.List;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.annotations.Test;
+
+public class GithubPageSeleniumTestsEx1 extends WebdriverSettings {
+
+    @Test
+    public void homePageTest() {
+        // 1. Open test site by URL
+        webDriver.navigate().to("https://jdi-testing.github.io/jdi-light/index.html");
+
+        // 2. Assert Browser title
+        soft.assertThat(webDriver.getTitle()).isEqualTo("Home Page");
+
+        // 3. Perform login
+        webDriverWait.until(ExpectedConditions.elementToBeClickable(
+            webDriver.findElement(By.xpath("//*[@class='profile-photo'][1]")))).click();
+        webDriver.switchTo().activeElement();
+        webDriverWait.until(ExpectedConditions.visibilityOf(
+            webDriver.findElement(By.xpath("//input[@id='name'][1]")))).sendKeys("Roman");
+        webDriverWait.until(ExpectedConditions.visibilityOf(
+            webDriver.findElement(By.xpath("//input[@id='password'][1]")))).sendKeys("Jdi1234");
+        webDriverWait.until(ExpectedConditions.elementToBeClickable(
+            webDriver.findElement(By.xpath("//button[@id='login-button'][1]")))).click();
+
+        // 4. Assert Username is loggined
+        String userName = webDriverWait.until(ExpectedConditions.visibilityOf(
+            webDriver.findElement(By.xpath("//span[@id='user-name']")))).getText();
+        soft.assertThat(userName).isEqualTo("ROMAN IOVLEV");
+
+        // 5. Assert that there are 4 items on the header section are displayed and they have proper texts
+        webDriverWait.until(ExpectedConditions.visibilityOfAllElements(
+                webDriver.findElements(By.xpath("//ul[@class='uui-navigation nav navbar-nav m-l8'][1]/li"))));
+        List<WebElement> menuTabs = webDriver.findElements(
+            By.xpath("//ul[@class='uui-navigation nav navbar-nav m-l8'][1]/li"));
+        soft.assertThat(menuTabs.size()).isEqualTo(4);
+        soft.assertThat(menuTabs.get(0).getText()).isEqualTo("HOME");
+        soft.assertThat(menuTabs.get(1).getText()).isEqualTo("CONTACT FORM");
+        soft.assertThat(menuTabs.get(2).getText()).isEqualTo("SERVICE");
+        soft.assertThat(menuTabs.get(3).getText()).isEqualTo("METALS & COLORS");
+
+        // 6. Assert that there are 4 images on the Index Page and they are displayed
+        webDriverWait.until(ExpectedConditions.visibilityOfAllElements(
+            webDriver.findElements(By.cssSelector(".benefit-icon"))));
+        List<WebElement> images = webDriver.findElements(By.cssSelector(".benefit-icon"));
+        soft.assertThat(images.size()).isEqualTo(4);
```

Comment:
Please use a specific matcher for collection size
```suggestion
        soft.assertThat(images).hasSize(4);
```